pdns/common_startup.cc

```diff
@@ -697,6 +697,12 @@ void mainthread()
 
   pdns::parseTrustedNotificationProxy(::arg()["trusted-notification-proxy"]);
 
+  try {
+    UeberBackend B;
+    B.updateZoneCache();
+  }
+  catch(...) {}
```

Comment:
that's a big catchall

Reply:
We can catch the same exceptions as in the loop and log them, but should we exit during the initial refresh?